configuration/verify-service-provider.yml

```diff
@@ -24,10 +24,9 @@ logging:
 clockSkew: ${CLOCK_SKEW:-PT5s}
 
 # Entity ID that uniquely identifies your service
-serviceEntityId: ${SERVICE_ENTITY_ID:-http://verify-service-provider}
-
-# Location where Verify Hub would send the assertions to
-assertionConsumerServiceUri: ${ASSERTION_CONSUMER_SERVICE_URI:-http://localhost:3200/verify/response}
+serviceEntityId: [${SERVICE_ENTITY_ID:-"http://verify-service-provider"}]
+# To use the Verify Service Provider with multiple services, add their entity IDs here, e.g.
+# ["service-one-entity-id", "service-two-entity-id"]
 
```

Comment:
I am slightly torn on leaving serviceEntityId singular, but it probably does make more sense for the mainline case of a single service.

Reply:
Could we have `serviceEntityId` and `serviceEntityIds` and error if both are present?

Reply:
This would be great. It turns out this is more complicated than I'm going to get done today, so I'm pushing an option which renames this to serviceEntityIds and just gets them to define a list, which may only have one element (because, on discussion, we thought that made more sense).

Should probably discuss having both - it would be nicer, but is probably more complicated config for users, even in the mainline case, and definitely more complicated code (it will be a bit messier in places, and we'll need to define our own validation thing sensibly which seems to be possible but more complicated than it originally sounded).

Reply:
A list that generally has only one entry sounds ok to me, especially if it makes it easier to explain both use cases